package/boot/uboot-envtools/patches/002-Revert-tools-env-use-run-to-store-lockfile.patch

```diff
@@ -0,0 +1,29 @@
+--- a/tools/env/fw_env_main.c
++++ b/tools/env/fw_env_main.c
```

Comment:
Please mention also in the patch description that this reverts https://source.denx.de/u-boot/u-boot/-/commit/aeb40f1166e072856f865d26d42a4bea1ec3a514.

Reply:
Added my original commit message to the patch, including the commit reference.